testdata/simple_expectation.json

```diff
@@ -4,12 +4,32 @@
     ],
     "name": "web",
     "owner": "self-hosted",
-    "properties": {
-        "prop_bool": "true",
-        "prop_empty_object": "{}",
-        "prop_object": "{\"message\":\"hello world\",\"condition\":true}",
-        "prop_string": "hello world"
-    },
+    "properties": [
```

Comment:
This change allows kubectl-opslevel and other client programs to just use `json.Unmarshal` to read a property, just like they can with a repository or tool.